### DISCUSSION TITLE
Multiline Input for Captures

### DISCUSSION THREAD

**Discussion by roccostorm** [chhoumann/quickadd#458] [Ideas]
Title says it all. I would like to have option for Multiline Input Field for Captures. 

Is this already possible (haven't found it)?

Thanks!

**Comment by chhoumann** [comment]
Hey @roccostorm 

There's a setting on the main settings page:
![image](https://user-images.githubusercontent.com/29108628/231735235-e0ac7bc1-1d70-4d64-a5a4-815628b20f92.png)

Is that what you're looking for?

**Reply by CarolusIII** [reply]
Multi line input prompt is great. Would really benefit if you could select this option on a per Choice basis. Sometimes the expected input is very small, like YES or OK and then multiline frame hides the Save button on ios. In such cases a single line would be better, so offer a choice  if possible. 
Thanks for all your good work.

**Comment by roccostorm** [comment]
Oh man... Yes, it's exactly what I'm loocking for.

Case closed...
